awsagentprovider/src/main/java/software/amazon/opentelemetry/javaagent/providers/AwsSpanProcessingUtil.java

```diff
@@ -40,6 +47,19 @@ final class AwsSpanProcessingUtil {
   static final String LOCAL_ROOT = "LOCAL_ROOT";
   static final String SQS_RECEIVE_MESSAGE_SPAN_NAME = "Sqs.ReceiveMessage";
   static final String AWS_SDK_INSTRUMENTATION_SCOPE_PREFIX = "io.opentelemetry.aws-sdk-";
+  // Max keyword length supported by parsing into remote_operation from DB_STATEMENT
```

Comment:
Can you also describe in the comment how we arrived at the value of the `MAX_KEYWORD_LENGTH` and if we ever need to update this value, how should we decide the new value?

Reply:
Added comment describing why we choose 27.